yarl/__init__.py

```diff
@@ -858,6 +858,8 @@ def _query_var(v):
             return v
         if type(v) is int:  # no subclasses like bool
             return str(v)
+        if isinstance(v, bool):
+            return "true" if v else "false"
```

Comment:
Thank you for the PR.

There is no the single representation for `bool` in the query string.
Some systems prefer `true`/`false`, others like `yes`/`no`, `on`/`off`, `Y`/`N`, `1`/`0`, etc.
That's why `yarl` doesn't make this decision and don't support `bool` explicitly. Please convert booleans to your preferred form explicitly before calling `yarl` API.